ui/user/src/routes/v2/admin/guardrails/+page.svelte

```diff
@@ -0,0 +1,172 @@
+		out:fly={{ x: -100, duration }}
+	>
+		{#if showCreateGuardrail}
+			{@render createGuardrailScreen()}
+		{:else}
+			<div
+				class="flex flex-col gap-8"
+				in:fly={{ x: 100, delay: duration, duration }}
+				out:fly={{ x: -100, duration }}
+			>
+				<div class="flex items-center justify-between">
+					<h1 class="text-2xl font-semibold">Guardrails</h1>
+					{#if guardrails.length > 0}
+						<div class="relative flex items-center gap-4">
+							{@render addGuardrailButton()}
+						</div>
+					{/if}
+				</div>
+				{#if guardrails.length === 0}
+					<div class="mt-12 flex w-md flex-col items-center gap-4 self-center text-center">
+						<TrainTrack class="size-24 text-gray-200 dark:text-gray-900" />
+						<h4 class="text-lg font-semibold text-gray-400 dark:text-gray-600">
+							No created guardrails
+						</h4>
+						<p class="text-sm font-light text-gray-400 dark:text-gray-600">
+							Looks like you don't have any guardrails created yet. <br />
+							Click the button below to get started.
+						</p>
+
+						{@render addGuardrailButton()}
+					</div>
+				{:else}
+					<Table
+						data={guardrails}
+						fields={['displayName', 'models']}
+						onSelectRow={(d) => {
+							handleNavigation(`/v2/admin/guardrails/${d.id}`);
+						}}
+						headers={[
+							{
+								title: 'Name',
+								property: 'displayName'
+							}
+						]}
+					>
+						{#snippet actions(d)}
+							<button
+								class="icon-button hover:text-red-500"
+								onclick={(e) => {
+									e.stopPropagation();
+									guardrailToDelete = d;
+								}}
+								use:tooltip={'Delete Guardrail'}
+							>
+								<Trash2 class="size-4" />
+							</button>
+						{/snippet}
+						{#snippet onRenderColumn(property, d)}
+							{#if property === 'models'}
+								{@const count = d.models.length}
+								{count ? count : '-'}
+							{:else}
+								{d[property as keyof typeof d]}
+							{/if}
+						{/snippet}
+					</Table>
+				{/if}
+			</div>
+		{/if}
+	</div>
+</Layout>
+
+{#snippet addGuardrailButton()}
+	<button
+		class="button-primary flex items-center gap-1 text-sm"
+		onclick={() => (showCreateGuardrail = true)}
+	>
+		<Plus class="size-4" /> Add New Guardrail
+	</button>
+{/snippet}
+
+{#snippet createGuardrailScreen()}
+	<div
+		class="h-full w-full"
+		in:fly={{ x: 100, delay: duration, duration }}
+		out:fly={{ x: -100, duration }}
+	>
+		<GuardrailForm>
+			{#snippet topContent()}
+				<div class="flex flex-wrap items-center">
+					<button
+						onclick={() => (showCreateGuardrail = false)}
+						class="button-text flex -translate-x-1 items-center gap-2 p-0 text-lg font-light"
+					>
+						<ChevronLeft class="size-4" />
+						Guardrails
+					</button>
+					<ChevronLeft class="mx-2 size-4" />
+					<span class="text-lg font-light">Create Guardrail</span>
+				</div>
+			{/snippet}
+		</GuardrailForm>
+	</div>
+{/snippet}
+
+<Confirm
+	msg="Are you sure you want to delete this filter?"
```

Comment:
Should this say:


```suggestion
	msg="Are you sure you want to delete this guardrail?"
```

?